CRM/Core/BAO/CustomField.php

```diff
@@ -1310,7 +1310,7 @@ private static function formatDisplayValue($value, $field, $entityId = NULL) {
             // In other contexts show a paperclip icon
             if (CRM_Utils_Rule::integer($value)) {
               $icons = CRM_Core_BAO_File::paperIconAttachment('*', $value);
-              $display = $icons[$value];
+              $display = $icons[$value] . civicrm_api3('File', 'getvalue', ['return' => "description", 'id' => $value]);
```

Comment:
Wondering if the getvalue api returns error incase File ID not found? 99% it shouldn't though.

Reply:
@pradpnayak I think its ok as $value as FileID will always carry a value (as per L1311 check) and then if the description not found then we should pass an empty string as in:
```php
$display = sprintf('%s %s',
  $icons[$value],
  civicrm_api3('File', 'getvalue', ['return' => "description", 'id' => $value]) ?: ''
 );
```